feat(changelog): generative-apis-added-new-models-support-llama-33 2025-01-09 (#4190)
scaleway/docs-content: changelog/january2025/2025-01-09-generative-apis-added-new-models-support-llama-33.mdx

---
title: New models support Llama 3.3
status: added
author:

>     fullname: ''

Reviewer: ```suggestion
    fullname: 'Join the #ai channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2025-01-09
category: ai-data
product: generative-apis
---

> [Llama 3.3 70B](https://www.scaleway.com/en/docs/ai-data/generative-apis/reference-content/supported-models/#request-a-model) is now available on Generative APIs.

Reviewer 2: ```suggestion
[Llama 3.3 70B](/ai-data/generative-apis/reference-content/supported-models/#request-a-model) is now available on Generative APIs.
```

> Llama 3.3 is a fine-tune of the Llama 3.1 70b model which was designed to approach the performance of Llama 3.1 405B on some applications.

Reviewer 3: ```suggestion
Llama 3.3 is a fine-tuned version of the Llama 3.1 70b model, which was designed to approach the performance of Llama 3.1 405B on some applications.
```
